data/xml/W18.xml

```diff
@@ -11338,7 +11338,7 @@
       <editor><first>Marco</first><last>Turchi</last></editor>
       <editor><first>Karin</first><last>Verspoor</last></editor>
       <publisher>Association for Computational Linguistics</publisher>
-      <address>Belgium, Brussels</address>
+      <address>Brussels, Belgium</address>
       <month>October</month>
```

Comment:
You think that's embarrassing? Then don't look at any of the footer stamps (for example, don't look at [this one](https://www.aclweb.org/anthology/W18-6301.pdf)).